presto-spi/src/main/java/com/facebook/presto/spi/eventlistener/TaskDistribution.java

```diff
@@ -44,7 +44,7 @@ public StageCpuDistribution(
             @JsonProperty("min") long min,
             @JsonProperty("max") long max,
             @JsonProperty("total") long total,
-            @JsonProperty("average") double average)
+            @JsonProperty("average") float average)
```

Comment:
This is unrelated change and therefore should go into separate commit. My preference would be to keep `double` though. Primarily to avoid having other readers wonder why float.

Reply:
Done